City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal updates for 01.06.2022

**Updates:**
```
- Upgrading composer/ca-bundle (1.3.1 => 1.3.2)
- Upgrading composer/composer (2.2.12 => 2.2.13)
- Upgrading composer/spdx-licenses (1.5.6 => 1.5.7)
- Upgrading consolidation/annotated-command (4.5.4 => 4.5.5)
- Upgrading drupal/coder (8.3.14 => 8.3.15)
- Upgrading drupal/core (9.3.12 => 9.3.14)
- Upgrading drupal/core-composer-scaffold (9.3.12 => 9.3.14)
- Upgrading drupal/core-dev (9.3.12 => 9.3.14)
- Upgrading drupal/core-recommended (9.3.12 => 9.3.14)
- Upgrading drupal/editoria11y (1.0.13 => 1.0.14)
- Upgrading drupal/gin (3.0.0-alpha37 => 3.0.0-beta3)
- Upgrading drupal/gin_toolbar (1.0.0-beta20 => 1.0.0-beta22)
- Upgrading drupal/helfi_tpr (2.0.11 => 2.1.1)
- Upgrading drupal/helfi_tunnistamo (2.0.6 => 2.0.7)
- Upgrading drupal/matomo (1.18.0 => 1.19.0)
- Upgrading drupal/migrate_plus (5.2.0 => 5.3.0)
- Upgrading drupal/oembed_providers (2.0.2 => 2.0.4)
- Upgrading drupal/pathauto (1.9.0 => 1.10.0)
- Upgrading guzzlehttp/guzzle (6.5.5 => 6.5.6)
- Upgrading josdejong/jsoneditor (v5.29.1 v5.29.1 => v5.29.1 )
- Upgrading league/uri (6.5.0 => 6.6.0)
- Upgrading nikic/php-parser (v4.13.2 => v4.14.0)
- Upgrading phpstan/phpdoc-parser (1.4.4 => 1.5.1)
- Upgrading ruflin/elastica (dev-master 72a4598 => dev-master 9c64c9c)
- Upgrading slevomat/coding-standard (7.1 => 7.2.1)
- Upgrading symfony/dom-crawler (v4.4.39 => v4.4.42)
- Upgrading symfony/filesystem (v4.4.39 => v4.4.42)
- Upgrading symfony/finder (v4.4.37 => v4.4.41)
- Upgrading symfony/phpunit-bridge (v5.4.7 => v5.4.8)
```

**To test**
- Checkout branch, run `make drush-cr drush-cim drush-updb`
- Check that all modules are up-to-date: https://helsinki-paatokset.docker.so/fi/admin/reports/updates
- Click around the site to see if anything breaks. There shouldn't be any major updates to the theme or functionalities
